Server/ConcordiaCurriculumManager/DTO/Courses/CourseDataDTO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ConcordiaCurriculumManager.Models.Curriculum;
+using System;
+namespace ConcordiaCurriculumManager.DTO.Courses
+{
+    public class CourseDataDTO
+    {

```

Comment:
We should also return Id, CourseNotes, SupportingFiles, and CourseCourseComponents instead of CourseComponents, so we can also have hours per component for the course (I believe that should be all fields). I'm not sure if you were up to date with main, so maybe you didnt see the new fields.

Reply:
Sounds good